electrumx/server/daemon.py

```diff
@@ -445,3 +445,15 @@ def client_session(self):
         # FIXME allow self signed certificates
         connector = aiohttp.TCPConnector(verify_ssl=False)
         return aiohttp.ClientSession(connector=connector)
+
+
+class PreLegacyRPCDaemon(LegacyRPCDaemon):
```

Comment:
Any reason you inherit LegacyRPCDaemon rather than Daemon?

Reply:
To manually recreate the block in bytes (as it is in the LegacyRPCDaemon)  as the getblock RPC call does not returns the block in hex